### PR TITLE
Change formatting/wording to be more explicit in express-middleware.mdx

### PR DESCRIPTION
**Pull request by RenierMeintjies**
Had some confusion reading this in the way it was formatted, any other suggestions welcome.

### CODE REVIEW

File: docs/source/api/express-middleware.mdx [hunk 1]
```diff
@@ -17,7 +17,7 @@ The `expressMiddleware` function expects you to set up HTTP body parsing and COR
 
 > See [Configuring CORS](../security/cors#specifying-origins) for guidance on configuring the CORS behavior of your project.
 
-The `expressMiddleware` function accepts two arguments. The first **required** argument is an instance of `ApolloServer` that has been `start`ed:
+The `expressMiddleware` function accepts two arguments. The first **required** argument is an instance of `ApolloServer` that has called the `start` method:
```
Comment (trevor-scheer) [inline docs/source/api/express-middleware.mdx:20]: `ApolloServer` doesn't call it, so this reads a bit off too. Here's my suggestion:
```suggestion
The `expressMiddleware` function accepts two arguments. The first **required** argument is an instance of `ApolloServer` that has been started by calling its `start` method:
```

**Review by trevor-scheer — CHANGES_REQUESTED** [review]
Thanks, just a small tweak to the wording.

File: docs/source/api/express-middleware.mdx (hunk 1 above)
Comment (RenierMeintjies) [reply docs/source/api/express-middleware.mdx]: Ok, awesome makes sense. Should I keep the change to the comments?

**Review by trevor-scheer — APPROVED** [review]
Thanks!

**Pull request merged**